modules/library/coverage/src/main/java/org/geotools/coverage/grid/io/imageio/geotiff/GeoTiffIIOMetadataDecoder.java

```diff
@@ -9,7 +9,7 @@
 
 The MIT License (MIT)
 
-Copyright (c) 2004 by Mike Nidel
+Copyright (c) 2004 - 2016 by Mike Nidel
```

Comment:
@georgekankava we also need to leave this one alone. This file also has an OSGeo copyright date range lower down that should be update to end in 2016. Sorry for the complication; only a few files in the code base are like this.